polygon/heimdall/milestone.go

```diff
@@ -72,15 +78,20 @@ func (m *Milestone) MarshalJSON() ([]byte, error) {
 }
 
 func (m *Milestone) UnmarshalJSON(b []byte) error {
+
+	// TODO - do we want to handle milestone_id ?
```

Comment:
what do you mean? what benefits doest it have? what if we don't handle it?

Reply:
I don't know  - hence the question. There seems to be a long hiemdall id sent on the json message.  I don't know what it is, it seems different from the id passed between erigon & heimdall.  I assume we can ignore it but am not 100% sure.

Reply:
I suggest to replace a TODO comment with a comment like: `ignoring an unused milestone_id, which looks like ...` to not bother us with thinking about whether or not TODO :) I feel that TODO is like a promise, and I prefer to follow a rule that all TODOs are either fixed, removed or moved to the task tracker in a limited time frame.